### PR TITLE
Update Mixer.md

### PR DESCRIPTION
**Pull request by TNgineering**
Added more details to the 'smix' procedure.

### CODE REVIEW

File: docs/Mixer.md [hunk 1]
```diff
@@ -99,21 +99,12 @@ Note: the `mmix` command may show a motor mix that is not active, custom motor m
...
-|----|--------------|
-| 0  | GIMBAL PITCH |
-| 1  | GIMBAL ROLL |
-| 2  | ELEVATOR / SINGLECOPTER_4 |
-| 3  | FLAPPERON 1 (LEFT) / SINGLECOPTER_1 |
-| 4  | FLAPPERON 2 (RIGHT) / BICOPTER_LEFT / DUALCOPTER_LEFT / SINGLECOPTER_2 |
-| 5  | RUDDER / BICOPTER_RIGHT / DUALCOPTER_RIGHT / SINGLECOPTER_3 |
-| 6  | THROTTLE (Based ONLY on the first motor output) |
-| 7  | FLAPS |
+The mmix statement has the following syntax: `smix n PWM_PIN SIGNAL_SOURCE RATE SPEED	MIN	MAX`
```
Comment (digitalentity) [inline docs/Mixer.md]: This is not PWM_PIN, this is servo ID. The actual PWM pins are board and mixer-specific.

**Review by digitalentity — CHANGES_REQUESTED** [review]

File: docs/Mixer.md (hunk 1 above)
Comment (TNgineering) [reply docs/Mixer.md]: Ok, but does iNav still assign servos incrementally starting with the lowest?

Comment (digitalentity) [reply docs/Mixer.md]: Yes, according to what servos are used. Painless360 did quite a good video explaining how servo indexes are assinged to PWM channels here https://youtu.be/qL2LMFF5bJ4

**Review by TNgineering — COMMENTED** [review]
I changed the syntax to refer to SERVO_ID instead of PWM_PIN

**Review by digitalentity — APPROVED** [review]

**Comment by digitalentity** [comment]
Thanks. Merged.

**Pull request merged**